projects/04-burger-queen-api-client/README.pt.md

```diff
@@ -147,8 +147,17 @@ Cumpra todas as histórias possíveis dentro do tempo especificado.
 A lógica do projeto deve ser totalmente implementada em JavaScript (ES6 +), HTML
 e CSS e empacotada de forma automatizada.
 
-Neste projeto você deve usar [React](https://pt-br.react.dev/) ou
-[Angular](https://angular.io/)).
+Um dos principais objetivos deste projeto é aprender a usar uma biblioteca ou
+framework popular para desenvolver uma aplicação web. Você deve escolher entre
+[React](https://pt-br.react.dev/) ou [Angular](https://angular.io/).
+
+Tenha em mente que se você escolher o Angular, é obrigatório usar [TypeScript](https://www.typescriptlang.org/).
+O _TypeScript_ é uma linguagem de programação fortemente tipada baseada
+em JavaScript.
+
+Se você optar pelo React, a decisão de usar o TypeScript é opcional (mas
+recomendamos!). Aqui você pode encontrar mais informações sobre
+como iniciar seu projeto com [Typescript e React](https://itnext.io/create-react-v18-typescript-project-with-vite-d0d602e4a60e).
```

Comment:
Ese link es `available to Medium members only` y no si puede verlo todo

Reply:
Creo podemos usar el link de documentacion https://vitejs.dev/guide/#scaffolding-your-first-vite-project que habla de los templates 
https://es.vitejs.dev/guide/#monta-tu-primer-proyecto-vite
https://pt.vitejs.dev/guide/#scaffolding-your-first-vite-project